cityofaustin/vision-zero
# Enable address editing

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19014

## Testing

**URL to test:** 
[Netlify](https://deploy-preview-1564--atd-vze-staging.netlify.app/#/crashes)

**Steps to test:**
1. Go to a crash
2. Test editing the primary address/secondary address fields
3. When you type into an input your input should be auto capitalized
4. If you are editing and click on the swap address button before saving/canceling, you will be taken out of edit mode on that field
5. Test swapping the addresses

---

#### Ship list
